db/fixtures/miq_policy_sets.yml

```diff
@@ -39,6 +39,21 @@
           description: Mark as Non-Compliant
           action_type: default
           options: {}
+      - qualifier: failure
+        failure_sequence: 2
+        failure_synchronous: true
+        MiqEventDefinition:
+          name: containerimage_compliance_check
+          description: Container Image Compliance Check
+          event_type: Default
+          definition:
+          default:
+          enabled:
+        MiqAction:
+          name: container_image_annotate_deny_execution
+          description: Prevent container image from running on OpenShift
+          action_type: default
```

Comment:
both searches are case sensitive (miq_action.rb and miq_event_definition.rb). I think changing one will require a migration.